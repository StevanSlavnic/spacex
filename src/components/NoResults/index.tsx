export default function NoResults() {
  return (
    <div className="flex items-center justify-center h-full">
      <div className="text-center">
        <h2 className="text-2xl font-semibold text-gray-700">
          No Results Found
        </h2>
        <p className="text-gray-500">
          Try adjusting your search or filter to find what you're looking for.
        </p>
      </div>
    </div>
  );
}
