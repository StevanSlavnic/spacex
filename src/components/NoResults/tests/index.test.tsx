import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import NoResults from "../index";

describe("NoResults Component", () => {
  it("should render without crashing", () => {
    const { container, getByText } = render(<NoResults />);
    expect(getByText("No Results Found")).toBeInTheDocument();
    expect(
      getByText(
        "Try adjusting your search or filter to find what you're looking for."
      )
    ).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
