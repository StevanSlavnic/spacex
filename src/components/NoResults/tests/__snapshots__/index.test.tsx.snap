// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults Component should render without crashing 1`] = `
<div>
  <div
    class="flex items-center justify-center h-full"
  >
    <div
      class="text-center"
    >
      <h2
        class="text-2xl font-semibold text-gray-700"
      >
        No Results Found
      </h2>
      <p
        class="text-gray-500"
      >
        Try adjusting your search or filter to find what you're looking for.
      </p>
    </div>
  </div>
</div>
`;
